src/ucp/core/ucp_ep.h

```diff
@@ -606,11 +606,11 @@ size_t ucp_ep_config_get_zcopy_auto_thresh(size_t iovcnt,
                                            const ucp_context_h context,
                                            double bandwidth);
 
-ucs_status_t ucp_worker_create_mem_type_endpoints(ucp_worker_h worker);
+ucs_status_t ucp_worker_create_mem_type_eps(ucp_worker_h worker);
 
-void ucp_worker_destroy_mem_type_endpoints(ucp_worker_h worker);
+void ucp_worker_destroy_mem_type_eps(ucp_worker_h worker);
```

Comment:
maybe ucp_worker_mem_type_eps_[create|destroy] for consistency with ucp_worker_mem_type_eps_print_info

Reply:
makes sense, done